### DISCUSSION TITLE
Double slash (//) in og:url

### DISCUSSION THREAD

**Discussion by followmyrandomness** [GetPublii/Publii#1863] [Bug Report]
### Operating system

Linux

### Publii version

0.46.2 (build 16966)

### Issue type

Application

### Bug description

As an example, the source of, say, https://demo.getpublii.eu/themes/square/design-thinking-for-innovative-ui/ contains:
`<meta property="og:url" content="https://demo.getpublii.eu/themes/square//design-thinking-for-innovative-ui/">`
with double slash after 'square'

**Comment by jeckyl** [comment]
I can confirm that I have the same problem.

**Comment by dziudek** [comment]
Thanks for reporting it - will be fixed in v.0.46.4
